src/ansible_navigator/actions/_actions.py

```diff
@@ -95,7 +98,10 @@ def run_interactive(package: str, action: str, *args: Any, **_kwargs: Any) -> An
     """Call the given action's run"""
     action_cls = get(package, action)
     app, interaction = args
-    return action_cls(app.args).run(app=app, interaction=interaction)
+    if hasattr(action_cls(app.args), "run"):
```

Comment:
This looks like a lot of copy-paste and repeat initialization of an in-memory object.

Reply:
good catch, yeah, shouldn't need to instantiate it in the conditional, or I guess I could ahead of time